pom.xml

```diff
@@ -21,7 +21,7 @@
     <parent>
         <groupId>io.confluent</groupId>
         <artifactId>common</artifactId>
-        <version>6.0.3</version>
+        <version>6.1.4</version>
```

Comment:
is this the latest 6.x.y?

Reply:
No this is not, but this fixes the issue, and this is what the pr was tested against. I'll stick to this version as part of this pr.